page/index.html

```diff
@@ -7,6 +7,61 @@
     <title>Start - Myndigheten för yrkeshögskolan</title>
   </head>
   <body>
- 
+    <header class="grid-container">
+      <nav>
+        <ul>
+          <li>Start</li>
+          <li>Om oss</li>
+          <li>Verksamhetsområden</li>
+          <li>Statistik</li>
+          <li>Övright</li>
+        </ul>
+        <input type="text" placeholder="Ange sökord...">
+        <button>Sök</button>
+
+      </nav>
+      <div class="calender">
+        <div><h3>Kommande viktiga datum</h3>
+        </div>
+        <div><h5>December</h5>
+        <p>18: Ansök öppnar för att bedriva 
+          teckenspråks utbildning</p>
+          <p>19:Ansök öppnar för att bedriva 
+            Skrivtolkutbildning utbildning </p>
+            <p>25: MYH konferans för företagare</p>
+            <p>28: Sista ansökningsdagen till  
+              Konst- och kulturutbildning</p>
+      </div>
+      </div>
+      <div class="textarea">
+        <div>
+          <h1>Välkomen till MYH!</h1>
+          <p>Myndigheten för yrkeshögskolans huvuduppdrag är 
+            att ansvara för frågor som rör yrkeshögskolan 
+            i Sverige och verka för att utbildningarna 
+            tillgodoser arbetslivets behov av kompetens. </p>
+            <button>OM OSS</button>
+        </div>
+        <div><h2>Funderar du på att <br> söka till Yrkeshogskola?</h2>
+          <p>Gå vidare till Yrkeshöskolan.se för mer information
+             kring skolor,utbildningar,viktiga datum m.m.</p>
+             <button>Yrkeshögskolan.se</button>
+        </div>
+      </div>
```

Comment:
Maybe this is an section or article instead of div, because it "for grouping together a single part of the page that constitutes one single piece of functionality"? https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Document_and_website_structure#HTML_layout_elements_in_more_detail

Reply:
Thanks for the feedback! I've changed it to a section.